core/client/README.md

```diff
@@ -0,0 +1,364 @@
+# Kwil Go Client
+
+This folder contains the Go language client for interacting with a Kwil RPC
+provider. Package `client` may be used to build a third-party application with
+the ability to:
+
+- Retrieve the status of a Kwil network.
+- List and retrieve Kuneiform schema deployed on a Kwil network.
+- Deploy and drop Kwil schema.
+- Execute mutative actions defined in a scheme.
+- Call read-only actions without a network transaction.
+- Run ad-hoc SQL queries.
+- Retrieve account information, such as balance and nonce.
+- Check the status and execution outcome of a network transaction.
+
+The `client` package is used by the `kwil-cli` application to provide these
+functions on the command line. Go applications may use the package directly.
+
+## Get the `core` Go Module
+
+The `client` package is part of the `core` Go sub-module of the `kwil-db` repository. To use the package in your Go application, add it as a `require` in your project's `go.mod`:
+
+```sh
+$ go get github.com/kwilteam/kwil-db/core
+go: downloading github.com/kwilteam/kwil-db/core v0.1.2
+go: downloading github.com/kwilteam/kwil-db v0.7.2
```

Comment:
how come main module is downloaded? maybe it's only for download?

Reply:
I wondered the same, concluded the same -- probably due to version control constraints it must download the main module when resolving submodules.  Maybe?

Reply:
yeah I guess.